static_src/actions/service_actions.js

```diff
@@ -146,10 +147,13 @@ const serviceActions = {
     return Promise.resolve(serviceInstance);
   },
 
-  errorCreateInstance(err) {
+  errorCreateInstance(error) {
+    const { response } = error;
+    const safeError = (response && response.data) || { code: 500 };
```

Comment:
Regarding the first one, I've seen it done like this with ternary operators. `condition ? happy case : sad case` but sounds good.